elixir/apps/domain/mix.exs

```diff
@@ -78,6 +75,7 @@ defmodule Domain.MixProject do
       # https://github.com/elixir-lang/elixir/issues/12777
       # Remove Swoosh from Domain once this is fixed.
       {:phoenix_swoosh, "~> 1.0"},
+      {:hackney, "~> 1.9"},
```

Comment:
Needed to fix https://github.com/firezone/firezone/actions/runs/7348790224/job/20007576526#step:18:16